### PR TITLE
COMMERCE-9794

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-9794

commit's fix is coming from this https://github.com/liferay-commerce/liferay-portal/pull/3150

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 1]
```diff
@@ -616,4 +616,124 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9794 | As a Admin I want to be able to view the Setting Tab on Order Item from a virtual product"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:619]:
```suggestion
	@description = "COMMERCE-9794 | As an Admin, I want to be able to view the Virtual Settings tab on a virtual product's Order Item details"
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 2]
```diff
@@ -616,4 +616,124 @@ definition {
...
+		task ("And a virtual product is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				gtin = "GTIN1",
+				manufacturerPartNumber = "MPN1",
+				productDescription = "Full Description",
+				productName = "Virtual Product",
+				productType = "Virtual",
+				shortDescription = "Short Description",
+				sku = "VIRTUALSKU");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:652]: @Brunno0liveira why do you need `gtin` and `manufacturerPartNumber` fields here?

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 3]
```diff
@@ -616,4 +616,124 @@ definition {
...
+				value1 = "Commerce_White.jpg");
+
+			Click(
+				key_filterOption = "Add",
+				locator1 = "AppBuilder#CHOOSE_SEARCH_FILTER_BY_OPTION");
+
+			CommerceProducts.updateProductVirtualActivationStatus(
+				key_contentType = "Base Information",
+				key_fieldLabel = "Activation Status",
+				status = "Completed");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:683]: There's already a macro that uploads a file for a virtual product: `CommerceProducts.updateProductVirtualConfiguration()`, please use this one. Adjust the macro to include the activation status change macro(`updateProductVirtualActivationStatus`). Thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@Brunno0liveira please have a look at the comments. Also, please change the commit message of your first commit, following our standard commit rules for locators. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro [hunk 4]
```diff
@@ -781,7 +781,7 @@ definition {
 		Select(
 			key_fieldLabel = ${key_fieldLabel},
 			locator1 = "Select#GENERIC_SELECT_FIELD",
-			value1 = ${status});
+			value1 = ${activationStatus});
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro:784]: No need to change this variable name, see comment below

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@Brunno0liveira please move the update macro commit before the test commit, thanks!

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**